common:
    lidar_topic:  "/livox/lidar"                  # From livox_ros_driver custom msg
    lidar_color_topic:  "/livox/color_lidar"      # Pub color pointcloud
    odom_topic:  "/Odometry"                             # From FASTLIO2
    camera_topic:  "/camera/color/image_raw"
    frame_id: "livox_frame"                                   # From livox_ros_driver frame id

mapping:
    extrinsicT: [ 0.0144457,  -0.999828,  0.0116559,  -0.115962,
                            -0.0142691,  -0.0118621,  -0.999828, -0.0215207,
                            0.999794, 0.0142769,  -0.014438,  -0.0131816,
                            0.0,  0.0,  0.0,  1]        #  lidar2camera 外参  From livox_camera_calib

    intrisicT: [ 913.197692871094, 0, 648.626220703125,
                         0, 913.52783203125, 358.518096923828,
                         0, 0, 1]           # camera 内参

    ditortion: [0, 0, 0, 0, 0]      # camera 畸变系数
